model-engine/model_engine_server/core/celery/app.py

```diff
@@ -504,7 +504,7 @@ def _get_backend_url_and_conf(
     elif backend_protocol == "s3":
         backend_url = "s3://"
         if aws_role is None:
-            aws_session = session(infra_config().profile_ml_worker)
+            aws_session = session(infra_config().profile_ml_inference_worker)
```

Comment:
Are we sure this doesn't break things?

Reply:
actually wait this probably breaks the endpoint builder on launch (endpoint builder on launch probably uses ml-worker), I think we need to look at `AWS_PROFILE` and use that value (something along those lines anyways)